openstack-manager-blueprint-inputs.yaml

```diff
@@ -188,50 +188,33 @@
 # Also note that the database username and password are hardcoded to root:root.
 #influxdb_endpoint_ip: ''
 
-
-###########################
+#############################
 # Offline Resources Upload
-###########################
+#############################
 # You can configure a set of resources to upload at bootstrap. These resources
-# will reside on the manager and enable offline deployment. dsl_resources
-# should contain any resource needed in the parsing process (i.e. plugin.yaml files),
-# and any plugin archive should be compiled through the designated wagon tool,
-# and the path should be passed to plugin_resources. Any resource your
-# blueprint might need, could be upload trough this mechanism.
+# will reside on the manager and enable offline deployment. `dsl_resources`
+# should contain any resource needed in the parsing process (i.e. plugin.yaml files)
+# and any plugin archive should be compiled using the designated wagon tool
+# which can be found at: http://github.com/cloudify-cosmo/wagon.
+# The path should be passed to plugin_resources. Any resource your
+# blueprint might need, could be uploaded using this mechanism.
+
 #dsl_resources:
 #  - {'source_path': '/cfy/cloudify/plugins/fabric-plugin/plugin.yaml', 'destination_path': '/fabric-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/plugins/script-plugin/plugin.yaml', 'destination_path': '/script-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/plugins/diamond-plugin/plugin.yaml', 'destination_path': '/diamond-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/plugins/openstack-plugin/plugin.yaml', 'destination_path': '/openstack-plugin/1.3rc1/plugin.yaml'}
-#  - {'source_path': '/cfy/cloudify/plugins/aws-plugin/plugin.yaml', 'destination_path': '/aws-plugin/1.3rc1/plugin.yaml'}
```

Comment:
talked with Idan, he does want all the IaaS plugins to be in the inputs section (commented out), both yaml and wagon wise.. personally im on the fence so it's up to you to decide whether you want to revert this part or postpone this decision to sunday :)